Create ui-c8y-1020-39-1-the-delete-popup-now-closes-correctly-if-another-delete-button-is.md (#2539)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-39-1-the-delete-popup-now-closes-correctly-if-another-delete-button-is.md

---
date: ""

> title: The delete popup now closes correctly if another delete button is hit. (#7339) (#7369)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fixed issue with delete popup not closing correctly when hitting another delete button in the bookmark edit view
```

> title: Fixed issue with delete popup not closing correctly when hitting another delete button in the bookmark edit view

Reviewer: ```suggestion
title: Delete popup now closing correctly when hitting another delete button in the bookmark edit view
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60830
version: 1020.39.1
---

> The delete popup now closes correctly if another delete button is hit. (#7339) (#7369)

Author: The Claude AI suggested to use the following body:
```suggestion
In certain scenarios, clicking on a delete button while the delete confirmation popup was already open would lead to unexpected behavior and the popup not closing properly. This issue has now been resolved. The delete confirmation popup will now always close correctly when clicking on another delete button, allowing users to smoothly continue their workflow without any interruptions. This change improves the overall user experience and consistency when working with delete confirmations in the bookmark edit view.
```

> In certain scenarios, clicking on a delete button while the delete confirmation popup was already open would lead to unexpected behavior and the popup not closing properly. This issue has now been resolved. The delete confirmation popup will now always close correctly when clicking on another delete button, allowing users to smoothly continue their workflow without any interruptions. This change improves the overall user experience and consistency when working with delete confirmations in the bookmark edit view.

Reviewer: Does this only apply to the "bookmark edit view" as stated in the last sentence, or to any "certain scenarios"? If it only applies to the "bookmark edit view", I would start with "In the bookmark edit view". If it applies to various screnarios, I would remove "in the bookmark edit view" from the last sentence.

Reviewer 2: This only applies to the bookmark edit view:
```suggestion
In the bookmark edit view, clicking on a delete button while the delete confirmation popup was already open would lead to unexpected behavior and the popup not closing properly. This issue has now been resolved. The delete confirmation popup will now always close correctly when clicking on another delete button, allowing users to smoothly continue their workflow without any interruptions. This change improves the overall user experience and consistency when working with delete confirmations.
```
